packages/investor-yearn/src/YearnOpportunity.ts

```diff
@@ -176,12 +177,16 @@ export class YearnOpportunity
     wallet: HDWallet
     tx: PreparedTransaction
     feePriority?: FeePriority
+    accountNumber?: number
```

Comment:
make this an optional `bip44Params`

Reply:
and throw if it's not provided - this makes the type sig backwards compatible and means that we know web is up to date

Reply:
Done in b1115cae7c1542776eb72c2d05dfb7853c332f41 - that also solves the problem of `accountNumber` being optional, since we will eventually add it to all investor packages and be able to make it required at `investor` level